src/makielayout/blocks/menu.jl

```diff
@@ -96,12 +102,136 @@ function initialize_block!(m::Menu)
         end
     end
 
-    selectiontext = Label(scene, selected_text, tellwidth = false, halign = :left,
-        padding = m.textpadding, textsize = m.textsize, color = m.textcolor)
+    selectionarea = Observable(Rect2f(0, 0, 0, 0))
+
+    selectionpoly = poly!(blockscene, selectionarea, color = m.selection_cell_color_inactive[])
```

Comment:
The way `DataInspector` is set up it just works with any plot that doesn't explicitly exclude itself. So plot calls in blocks should have `inspectable = false` to be hidden

Reply:
I guess that makes sense, but shouldn't DataInspector be axis specific?

Reply:
I didn't want it to be coupled to Axis directly when I made it. I'd rather have it work with anything.

Maybe we could make it look for a certain parent scene. I.e. effectively check `parent_scene(hovered_plot) == ax.blockscene.childrenb[1]` after initialising with `DataInspector(ax)`. That would probably need to be a recursion to still allow it to work with any plot after `DataInspector(fig)` though.

Reply:
Well, seems kind of important, because you can't just expect anyone creating a recipe/whatnot foresee if someone might want to inspect it at some point... or not ;)

Reply:
I have to add, just from an end-user's perspective, if I want to make one plot inspectable it's a bit of a hassle then having to add `inspectable = false` to *every other component of the figure*.